clingwrapper/src/clingwrapper.cxx

```diff
@@ -151,14 +151,31 @@ static struct Signalmap_t {
 //     }
 // };
 
+static inline std::vector <const char*> split_by_spaces(char *s) {
+    std::vector <const char *> tokens;
+    char *token = strtok(s, " ");
+
+    while (token) {
+        tokens.push_back(token);
+        token = strtok(NULL, " ");
+    }
+    return tokens;
+}
+
 class ApplicationStarter {
   InterOp::TInterp_t Interp;
 public:
     ApplicationStarter() {
         // Create the interpreter and initilize the pointer
         // FIXME: We should make these flags available via the env variable
         // INTEROP_EXTRA_INTERPRETER_ARGS
-         Interp = InterOp::CreateInterpreter({"-std=c++17", "-march=native"});
+        char *InterpArgString = getenv("INTEROP_EXTRA_INTERPRETER_ARGS");
+
+        std::vector <const char *> InterpArgs = InterpArgString ?
+                split_by_spaces(InterpArgString) :
+                std::vector<const char *>({"-std=c++17", "-march=native"});
```

Comment:
We should prepend these to `InterpArgs`. Then when we append the flags from `INTEROP_EXTRA_INTERPRETER_ARGS` they will overwrite them if the user chose something else.

Reply:
Sure, fixed.